drivers/wifi/nrf700x/src/fw_load.c

```diff
@@ -89,7 +95,7 @@ enum nrf_wifi_status nrf_wifi_fw_load(void *rpu_ctx)
 		LOG_ERR("%s: nrf_wifi_fmac_fw_load failed", __func__);
 	}
 
-#if defined(CONFIG_NRF_WIFI_PATCHES_EXT_FLASH) && defined(CONFIG_NORDIC_QSPI_NOR)
+#if defined(CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP) && defined(CONFIG_NORDIC_QSPI_NOR)
```

Comment:
How about making `NRF_WIFI_PATCHES_EXT_FLASH_XIP` dependent on `NORDIC_QSPI_NOR` in Kconfig if both are needed?